docs/member/issuer-profile.md

```diff
@@ -0,0 +1,78 @@
+# Issuer Profile
+
+## 1. What is an Issuer?
+
+A legal entity that manages, registers, and sells securities to support its operations financially is an Issuer. There can be many types of Issuers - Corporations, Domestic or Foreign Governments and Investment trusts. Issuers develop securities such as Bonds, Debentures, G-Secs & NCDS. Issuers are legally responsible for reporting financial conditions, material developments, and other operational activities required by the jurisdiction's regulations.
```

Comment:
`NCDS`

check if the last letter S is a part of the NCD acronym or it is a plural form.

if it is a plural form then S should be in small caps, like NCDs
if it is a part of the acronym, then this is correct

Reply:
It is plural, changes made